### DISCUSSION TITLE
openknowledge/ckan-base: X.X  missing variable

### DISCUSSION THREAD

**Discussion by ghost** [ckan/ckan#7335] [General]
Hello, 

While building a docker process i have noticed that in openknowledge/ckan-dev  there is a variable to pick up extensions which is not present in the ckan-base images. 

ENV SRC_EXTENSIONS_DIR=/srv/app/src_extensions

is there a work around or can the image be updated?

**Comment by amercader** [comment]
For questions about [`okfn/docker-ckan`](https://github.com/okfn/docker-ckan) is better to use that particular repo.
The `SRC_EXTENSIONS_DIR` var is used to point to a folder in the image that will be mounted in the host to allow to work on extensions locally which is something that only happens in the dev images. For the production ckan-base images the var is not present because that mounting doesn't happen

**Reply by ghost** [reply]
Thanks for the response,  

how come when I run the code against ckan-base (rather than ckan-dev) the extensions cant be found - do I need to move them from the src folder?
